### DISCUSSION TITLE
Integration of 3D mousse

### DISCUSSION THREAD

**Discussion by Zahorone** [rustdesk/rustdesk#9528] [Feature Request]
Is it planned integration 3D mouse (for example 3Dconnexion) to work with CAD applications on client PC? Both computers installed the same drivers.

**Comment by 3Duke** [comment]
me too

**Comment by Macualey3** [comment]
I don't understand how not a single remote desktop software has this ability. Literally one of the most important features for me.

**Comment by 17o2** [comment]
Add me to the potential list of users :)
